### PR TITLE
Gh 467 seed hierarchy

### PR DESCRIPTION
**Pull request by p013570**
Sorry another big pull request. I have basically replaced all uses of ElementSeed with ElementId.

### CODE REVIEW

File: core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/get/GetAdjacentIds.java [hunk 1]
```diff
@@ -31,20 +31,19 @@
 import java.util.Map;
 
 /**
- * An <code>GetAdjacentEntitySeeds</code> operation will return the
- * {@link uk.gov.gchq.gaffer.operation.data.EntitySeed}s at the opposite end of connected edges to a seed
- * {@link uk.gov.gchq.gaffer.operation.data.EntitySeed}.
+ * An <code>GetAdjacentVertices</code> operation will return the
```
Comment (gaffer01) [inline core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/get/GetAdjacentIds.java]: This should say GetAdjacentIds

File: core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/get/GetAdjacentIds.java [hunk 2]
```diff
@@ -31,20 +31,19 @@
...
+ * vertices at the opposite end of connected edges to a provided seed vertices
```
Comment (gaffer01) [inline core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/get/GetAdjacentIds.java]: "vertices" should be "vertex."

**Comment by gaffer01** [comment]
I spotted a couple of typos so added a note about them. Otherwise, looks good, but worth @t616178 checking as well as it's a big change.

**Comment by p013570** [comment]
Thanks @gaffer01 I'll get them fixed.

**Comment by p013570** [comment]
I've made the requested changes.

**Pull request merged**